ui/src/frame.tsx

```diff
@@ -104,8 +110,8 @@ export const XFrame = ({ model: { name, path, content, width = '100%', height =
 export const
   View = bond(({ name, state, changed }: Model<State>) => {
     const render = () => (
-      <div data-test={name} className={css.card}>
-        <div className='wave-s12 wave-w6'>{state.title}</div>
+      <div data-test={name} className={clas(css.card, (state.compact ?? false) ? '' : css.cardPadding)}>
```

Comment:
nit: It's going to be falsy by default anyway so this should be enough.
```suggestion
      <div data-test={name} className={clas(css.card, state.compact ? '' : css.cardPadding)}>
```